digraph "unitGraph" {
    "z0 = 0"
    "$r0 = newarray (byte)[8]"
    "$r0[0] = 90"
    "$r0[1] = 91"
    "$r0[2] = 92"
    "$r0[3] = 93"
    "$r0[4] = 94"
    "$r0[5] = 95"
    "$r0[6] = 96"
    "$r0[7] = 97"
    "r1 = $r0"
    "$r2 = new java.util.zip.InflaterInputStream"
    "$r3 = new java.io.ByteArrayInputStream"
    "specialinvoke $r3.<java.io.ByteArrayInputStream: void <init>(byte[])>(r1)"
    "specialinvoke $r2.<java.util.zip.InflaterInputStream: void <init>(java.io.InputStream)>($r3)"
    "r4 = $r2"
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void mark(int)>(-5)"
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void mark(int)>(6)"
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void mark(int)>(2147483647)"
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void reset()>()"
    "$r5 = new java.lang.RuntimeException"
    "specialinvoke $r5.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Expected exception not thrown\")"
    "throw $r5"
    "$r7 := @caughtexception"
    "r9 = $r7"
    "$r8 = new java.lang.RuntimeException"
    "specialinvoke $r8.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Expected exception not thrown\")"
    "throw $r8"
    "$r6 := @caughtexception"
    "r10 = $r6"
    "return"
    "z0 = 0"->"$r0 = newarray (byte)[8]";
    "$r0 = newarray (byte)[8]"->"$r0[0] = 90";
    "$r0[0] = 90"->"$r0[1] = 91";
    "$r0[1] = 91"->"$r0[2] = 92";
    "$r0[2] = 92"->"$r0[3] = 93";
    "$r0[3] = 93"->"$r0[4] = 94";
    "$r0[4] = 94"->"$r0[5] = 95";
    "$r0[5] = 95"->"$r0[6] = 96";
    "$r0[6] = 96"->"$r0[7] = 97";
    "$r0[7] = 97"->"r1 = $r0";
    "r1 = $r0"->"$r2 = new java.util.zip.InflaterInputStream";
    "$r2 = new java.util.zip.InflaterInputStream"->"$r3 = new java.io.ByteArrayInputStream";
    "$r3 = new java.io.ByteArrayInputStream"->"specialinvoke $r3.<java.io.ByteArrayInputStream: void <init>(byte[])>(r1)";
    "specialinvoke $r3.<java.io.ByteArrayInputStream: void <init>(byte[])>(r1)"->"specialinvoke $r2.<java.util.zip.InflaterInputStream: void <init>(java.io.InputStream)>($r3)";
    "specialinvoke $r2.<java.util.zip.InflaterInputStream: void <init>(java.io.InputStream)>($r3)"->"r4 = $r2";
    "r4 = $r2"->"virtualinvoke r4.<java.util.zip.InflaterInputStream: void mark(int)>(-5)";
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void mark(int)>(-5)"->"virtualinvoke r4.<java.util.zip.InflaterInputStream: void mark(int)>(6)";
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void mark(int)>(6)"->"virtualinvoke r4.<java.util.zip.InflaterInputStream: void mark(int)>(2147483647)";
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void mark(int)>(2147483647)"->"virtualinvoke r4.<java.util.zip.InflaterInputStream: void reset()>()";
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void mark(int)>(2147483647)"->"$r7 := @caughtexception";
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void reset()>()"->"$r5 = new java.lang.RuntimeException";
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void reset()>()"->"$r7 := @caughtexception";
    "$r5 = new java.lang.RuntimeException"->"specialinvoke $r5.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Expected exception not thrown\")";
    "$r5 = new java.lang.RuntimeException"->"$r7 := @caughtexception";
    "specialinvoke $r5.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Expected exception not thrown\")"->"throw $r5";
    "specialinvoke $r5.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Expected exception not thrown\")"->"$r7 := @caughtexception";
    "throw $r5"->"$r7 := @caughtexception";
    "$r7 := @caughtexception"->"r9 = $r7";
    "r9 = $r7"->"virtualinvoke r4.<java.util.zip.InflaterInputStream: void reset()>()";
    "r9 = $r7"->"$r6 := @caughtexception";
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void reset()>()"->"$r8 = new java.lang.RuntimeException";
    "virtualinvoke r4.<java.util.zip.InflaterInputStream: void reset()>()"->"$r6 := @caughtexception";
    "$r8 = new java.lang.RuntimeException"->"specialinvoke $r8.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Expected exception not thrown\")";
    "$r8 = new java.lang.RuntimeException"->"$r6 := @caughtexception";
    "specialinvoke $r8.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Expected exception not thrown\")"->"throw $r8";
    "specialinvoke $r8.<java.lang.RuntimeException: void <init>(java.lang.String)>(\"Expected exception not thrown\")"->"$r6 := @caughtexception";
    "throw $r8"->"$r6 := @caughtexception";
    "$r6 := @caughtexception"->"r10 = $r6";
    "r10 = $r6"->"return";
}
